src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -353,3 +353,28 @@ databaseChangeLog:
                   name: closed_issue_count
                   type: bigint
                   remarks: The number of existing issues closed by this request (if it was successfully processed at all).
+  - changeSet:
+      id: users-table
+      author: tim.best@usds.dhs.gov
+      comment: information about the users of the case-issue-api
+      changes:
+        - createTable:
+            tableName: user
+            remarks: Data about the users that login to the api.
+            columns:
+              - column: *pk_column
```

Comment:
Proposed alternative (in pseudo SQL):

   USER_ID varchar(255) PRIMARY KEY,
   USER_PRINT_NAME varchar(255) NOT NULL,
   USER_LAST_SEEN timestamp with time zone NOT NULL default current_timestamp